import org.apache.hadoop.io.retry.MultiException;
public class FunctionUse494 {
public void funcUse() {
MultiException multiexception = new MultiException();
multiexception.getExceptions();
}
}